vendor/README.md

```diff
@@ -137,7 +137,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **GAP:** [dhowden/gap-tmbundle](https://github.com/dhowden/gap-tmbundle)
 - **GCC Machine Description:** [textmate/lisp.tmbundle](https://github.com/textmate/lisp.tmbundle)
 - **GDB:** [quarnster/SublimeGDB](https://github.com/quarnster/SublimeGDB)
-- **GDScript:** [beefsack/GDScript-sublime](https://github.com/beefsack/GDScript-sublime)
+- **GDScript:** [PrestonKnopp/language-gdscript](https://github.com/PrestonKnopp/language-gdscript)
```

Comment:
This new grammar doesn't have a license, hence the test failure. We can't accept a grammar without a license.

Reply:
Okay, I missed that. I have created an issue there.